src/cclib/io/filewriter.py

```diff
@@ -89,6 +92,13 @@ def _make_bond_connectivity_from_openbabel(self, obmol):
                                         obbond.GetBondOrder()))
         return bond_connectivities
 
+    def _check_required_attributes(self, required):
```

Comment:
`required` should be a class variable, so you don't need to pass it to this method.